stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -59,14 +59,16 @@ public type Client client object {
     #
     # + sqlQuery - SQL statement to execute
     # + parameters - The parameters to be passed to the update query
+    # + getGeneratedKeys - Allow to retrieves auto-generated keys after a statement has been executed if value is
+    #                      `true`.It will return auto-generated keys only if supported by JDBC
     # + return - `UpdateResult` with the updated row count and key column values,
     #             else `Error` will be returned if there is an error
-    public remote function update(@untainted string sqlQuery, Param... parameters)
-                                  returns UpdateResult|Error {
+    public remote function update(@untainted string sqlQuery, boolean? getGeneratedKeys = (),
```

Comment:
IMO, instead of `boolean?` we should use `boolean` with default value. So for simple update statements which doesn't have parameters, we can specify the `sqlQuery` only